src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -347,3 +347,63 @@
     type: command
     short-summary: Update an Azure Cache for Redis service binding of the app.
 """
+
+helps['spring-cloud certificate add'] = """
+    type: command
+    short-summary: Add a certificate in Azure Spring Cloud.
+    examples:
+    - name: Import certificate from key vault.
+      text: az spring-cloud certificate add --name MyCertName --vault-uri MyKeyVaultUri --vault-certificate-name MyKeyVaultCertName
+"""
+
+helps['spring-cloud certificate show'] = """
+    type: command
+    short-summary: Show a certificate in Azure Spring Cloud.
+"""
+
+helps['spring-cloud certificate list'] = """
+    type: command
+    short-summary: List all certificates in Azure Spring Cloud.
+    examples:
+    - name: List all certificates in spring cloud service.
+      text: az spring-cloud certificate list -o table
+"""
+
+helps['spring-cloud certificate remove'] = """

```

Comment:
```
spring-cloud certificate remove
spring-cloud app custom-domain delete
```
I think the verb should be consistent. I prefer "delele"

Reply:
We should use those verbs in pair right? add/remove, create/delete, bind/unbind

Reply:
OK.

Reply:
So I would suggest:
```
spring-cloud certificate remove
spring-cloud app custom-domain unbind
```